skbeam/core/utils.py

```diff
@@ -1078,7 +1078,7 @@ def q_to_twotheta(q, wavelength):
     Returns
```

Comment:
Likewise I can't quite reach line 1073 to comment on it, but IIRC `q` here needs to be in inverse angstroms, right?

Reply:
As above nm^-1 is also viable.